# Задание 1

Вы провели эксперимент c упрощением формы заказа в магазине Утконос и получили результаты по метрике конверсий в покупку. 

Выберите метод оценки и оцените есть ли стат.значимые различия между конверсиями в двух группах при alpha = 5%. 

Дайте краткие рекомендации команде.

Результаты: 

    1 Число юзеров в группах, которые заходили на сайт в период эксперимента: n1 = 15550 и n2 = 15550. 

    2 Число юзеров в группах, которые совершили хотя бы одну покупку за период эксперимента: n1 = 164 и n2 = 228 

    3 Конверсии : conv1 = 1.05% conv2 = 1.47% .

In [11]:
n1 = 164
n2 = 228

N1 = 15550
N2 = 15550

p1 = n1 / N1
p2 = n2 / N2


In [12]:
p =  (n1 * p1 + n2 * p2) / (n1 + n2)

In [13]:
Z = (p1 - p2) / (p * (1 - p) * (1/N1 + 1/N2))**0.5
Z

-3.2110930329358367

Критическое значение Z-критерия при α=5% и двухсторонней альтернативе равно 1.96.

Так как расчетное значение Z меньше критического значения, т.е. -3.2110930329358367 < 1.96,
то мы отвергаем нулевую гипотезу о равенстве конверсий в группах.

    То есть, упрощённая форма заказа реально влияет на покупки в магазине Утконос

        Крайне, рекомендуется использовать новую форму заказа.

# Задание 2

Сравниваем метрику конверсия в покупку. Размер выборки - 10000 элементов в каждой группе.

Какой статистический критерий тут лучше всего подойдёт и почему ?

## Ответ:

при больших выборках (обычно >30), то нужно использовать Z-критерий.